Makefile

```diff
@@ -0,0 +1,104 @@
+# Verna's Makefile
+#
+# /!\ /!\ /!\ /!\ /!\ /!\ /!\ DISCLAIMER /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\
+#
+# This Makefile is only meant to be used for DEVELOPMENT purpose as we are
+# changing the user id that will run in the container.
+#
+# PLEASE DO NOT USE IT FOR YOUR CI/PRODUCTION/WHATEVER...
+#
+# /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\
+#
+# Note to developers:
+#
+# While editing this file, please respect the following statements:
+#
+# 1. Every variable should be defined in the ad hoc VARIABLES section with a
+#    relevant subsection
+# 2. Every new rule should be defined in the ad hoc RULES section with a
+#    relevant subsection depending on the targeted service
+# 3. Rules should be sorted alphabetically within their section
+# 4. When a rule has multiple dependencies, you should:
+#    - duplicate the rule name to add the help string (if required)
+#    - write one dependency per line to increase readability and diffs
+# 5. .PHONY rule statement should be written after the corresponding rule
+
+# ==============================================================================
+# VARIABLES
+
+# -- Docker
+# Get the current user ID to use for docker run and docker exec commands
+DOCKER_UID           = $(shell id -u)
+DOCKER_GID           = $(shell id -g)
+DOCKER_USER          = $(DOCKER_UID):$(DOCKER_GID)
+COMPOSE              = DOCKER_USER=$(DOCKER_USER) docker-compose
+COMPOSE_RUN          = $(COMPOSE) run --rm
+COMPOSE_EXEC         = $(COMPOSE) exec
+COMPOSE_EXEC_NODE    = $(COMPOSE_EXEC) node
+COMPOSE_RUN_CROWDIN  = $(COMPOSE_RUN) crowdin crowdin
+
+# -- Node
+# We must run node with a /home because yarn tries to write to ~/.yarnrc. If the
+# ID of our host user (with which we run the container) does not exist in the
+# container (e.g. 1000 exists but 1009 does not exist by default), then yarn
+# will try to write to "/.yarnrc" at the root of the system and will fail with a
+# permission error.
+COMPOSE_RUN_NODE     = $(COMPOSE_RUN) -e HOME="/tmp" node
+YARN                 = $(COMPOSE_RUN_NODE) yarn
+
+# ==============================================================================
+# RULES
+
+default: help
+
+# -- Development
+install: ## install dependencies
```

Comment:
`install` rule should be placed after build one as it is requested to be ordered in alphabetical one at the top of the file